index.html

```diff
@@ -1,25 +1,36 @@
 <!DOCTYPE HTML>
 <html lang="en">
     <head>
-        <meta charset="UTF-8">
```

Comment:
I see you made unrelated changes to the markup here, but it makes it harder to see what you added.

Reply:
Yes, that was bad practice. I moved the charset tag back to the original position.